web/src/modules/topic/components/Score/ScorePie.tsx

```diff
@@ -30,28 +43,33 @@ export const ScorePie = ({ pieDiameter, arguableId, arguableType }: Props) => {
   });
 
   return (
-    <Box display="flex" width={pieDiameter} height={pieDiameter}>
+    <StyledBox display="flex" width={pieDiameter} height={pieDiameter}>
       <PieChart
         data={data}
-        radius={50} // is relative to viewbox size [100, 100]
+        radius={45} // is relative to viewbox size [100, 100]
         label={({ dataEntry }) => dataEntry.key}
         labelStyle={() => ({
           fontSize: "14", // is relative to viewbox size [100, 100]
           pointerEvents: "none",
+          webKitUserSelect: "none" /* Safari */,
+          msUserSelect: "none" /* IE 10 and IE 11 */,
+          userSelect: "none" /* Standard syntax */,
         })}
         segmentsStyle={{
+          transformOrigin: "center",
```

Comment:
was this necessary? mozilla says this is default https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin